HotelBooking/Database/create_database.sql

```diff
@@ -43,6 +43,7 @@ CREATE TABLE reservation
   bill_id INTEGER,
   reservation_checkin_date TEXT,
   reservation_stay_date INTEGER,
+  is_accessibility_requested BIT,
```

Comment:
I double checked the docs for SQLite and SQLite does not support BOOLEAN, have you verified that the SQL library we are using, `SQLModel` does not auto convert Python Boolean to SQLite INT?

https://sqlmodel.tiangolo.com/tutorial/create-db-and-table/?h=boolean#text-or-varchar

Reply:
BIT takes values 0, 1, or Null and that is what is often used for booleans since no boolean support... 
1=true
0=false